dev-docs/bidders/nexverse.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: bidder
+title: Nexverse
+description: Prebid Nexverse Bidder Adapter
+biddercode: nexverse
+aliasCode: nexverse
```

Comment:
please remove the aliasCode unless you're aliasing another adapter, which I don't think you are.